src/components/greetings.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const greetings = () => (
+  <div>greetings</div>
+);
```

Comment:
You should make changes to this component as it is the component responsible for displaying the greetings fetched from the API.